diffs/nextStakedAave.md

```diff
@@ -1,34 +1,34 @@
+| ______gap                                  | uint256[50]                                                                         | 10   | 0      | 1600  | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| assets                                     | mapping(address => struct AaveDistributionManager.AssetData)                        | 60   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| stakerRewardsToClaim                       | mapping(address => uint256)                                                         | 61   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| stakersCooldowns                           | mapping(address => struct IStakedTokenV2.CooldownSnapshot)                          | 62   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| _votingDelegates                           | mapping(address => address)                                                         | 63   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| deprecated_propositionPowerSnapshots       | mapping(address => mapping(uint256 => struct GovernancePowerWithSnapshot.Snapshot)) | 64   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| deprecated_propositionPowerSnapshotsCounts | mapping(address => uint256)                                                         | 65   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| deprecated_propositionPowerDelegates       | mapping(address => address)                                                         | 66   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| _______DEPRECATED_DOMAIN_SEPARATOR         | bytes32                                                                             | 67   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| _nonces                                    | mapping(address => uint256)                                                         | 68   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| _admins                                    | mapping(uint256 => address)                                                         | 69   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| _pendingAdmins                             | mapping(uint256 => address)                                                         | 70   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| _votingDelegatee                           | mapping(address => address)                                                         | 71   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
+| _propositionDelegatee                      | mapping(address => address)                                                         | 72   | 0      | 32    | src/flattened/StakedAaveV3Flattened.sol:StakedAaveV3 |
```

Comment:
most likely should be deprecated as well, because existing delegations will be wiped

Reply:
This is declared in BaseDelegation Contract. even if it gets reset, i dont think it should be marked as deprecated, as it will still be used